engine/runtime/src/main/java/org/enso/interpreter/Language.java

```diff
@@ -68,6 +69,9 @@
    */
   @Override
   protected Context createContext(Env env) {
+    boolean logMasking = env.getOptions().get(RuntimeOptions.LOG_MASKING_KEY);
+    Masking$.MODULE$.setup(logMasking);
```

Comment:
Accesses like this can be a bit finnicky in Java. I'd suggest providing an acessor such as `Masking.getModule().setup(logMasking)`.